test/util/display_SYSINFO_settings.py

```diff
@@ -1,31 +1,28 @@
-# Copyright 2016, EMC, Inc.
+'''
+Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
```

Comment:
@gpaulos Did you do a git mv or a copy?  it looks like the files got added as new instead of just moved?